src/check/parse/tokenize.zig

```diff
@@ -11,11 +11,10 @@ const tracy = @import("../../tracy.zig");
 /// an index into the string interner
 pub const Token = struct {
     tag: Tag,
-    offset: u32,
```

Comment:
this offset is the same as offset of interned. we also checked if they are eq in some places. I suggest `interned` to be source of truth